packages/dd-trace/src/llmobs/plugins/openai/index.js

```diff
@@ -380,6 +405,21 @@ class OpenAiLLMObsPlugin extends LLMObsPlugin {
 
     this._tagger.tagLLMIO(span, inputMessages, outputMessages)
 
+    // Handle prompt tracking for reusable prompts
+    if (inputs.prompt && response && response.prompt) {
+      const { id, version } = response.prompt // ResponsePrompt
+      if (id && version) {
+        const normalizedVariables = normalizePromptVariables(inputs.prompt.variables)
+        const chatTemplate = extractChatTemplateFromInstructions(response.instructions, normalizedVariables)
+        this._tagger._setTag(span, '_ml_obs.meta.input.prompt', {
```

Comment:
i don't think i've made a ticket yet, but feel free to make one and add a TODO here that we'll follow up on turning this into an actual tagger API for future prompts work!